bridge/discord/webhook.go

```diff
@@ -84,7 +84,7 @@ func (b *Bdiscord) webhookSend(msg *config.Message, channelID string) (*discordg
 			}
 			content := fi.Comment
 
-			_, e2 := b.transmitter.Send(
+			res, err = b.transmitter.Send(
```

Comment:
Hypothetically we will overwrite a result and error from a sent message from line 63. 
However, this should not be possible as the text for a message sent stays in the file caption not the `msg.Text`

Reply:
This might depend on the bridge that created the `config.Message`, and produce an issue for bridges that actually do set `msg.Text`.

I think we need a better one-to-many relationship here, instead of assuming that 1 message on the sending-bridge side corresponds to 1 message on the receiving-bridge side.

Reply:
Yeah in that regard this would be an incomplete fix for the existing issue.

Also if there are multiple files in the Extra["file"] then we'd also only return the reference of the last file sent.
In order to fix this we'd have to refractor this all send methods to return an array of message id's rather than a single entry, something that would require some larger refactoring.

Reply:
A quick (dirty) fix is to 

```suggestion
			res2, err := b.transmitter.Send(
			...
			if msg.Text == "" {
			    res = res2
			}
```
This needs to be handled better but needs quite a refactoring to have this done correctly, but if this fixes some of the issues it's fine by me.